test/test_integration.py

```diff
@@ -421,12 +422,25 @@ def test_export_quantify_scheduler() -> None:
             "Rz(11.25, 'q[0]')",
             "Rxy(-90, 90, 'q[0]')",
             "Rz(-180, 'q[0]')",
+            "Measure q[0]",
             "Rz(-180, 'q[1]')",
             "Rxy(90, 90, 'q[1]')",
-            "Measure q[0]",
             "Measure q[1]",
         ]
 
+        ir_measurements = [instruction for instruction in circuit.ir.statements if isinstance(instruction, Measure)]
+        qs_measurements = [
+            operation.data["gate_info"]
+            for operation in exported_schedule.operations.values()
+            if operation.data["gate_info"]["operation_type"] == "measure"
+        ]
+
+        for i, ir_measurement in enumerate(ir_measurements):
+            qubit_index = ir_measurement.qubit.index
+            assert qs_measurements[i]["acq_channel_override"] == qubit_index
+            assert qs_measurements[i]["acq_index"] == qubit_index
+            assert qs_measurements[i]["acq_protocol"] == "ThresholdedAcquisition"
```

Comment:
I would probably wouldn't use a `qubit_index` var here either. But it's OK with me to leave it.